src/main/scala/is/hail/expr/ir/Copy.scala

```diff
@@ -53,9 +53,9 @@ object Copy {
       case ArrayRange(_, _, _) =>
         val IndexedSeq(start: IR, stop: IR, step: IR) = newChildren
         ArrayRange(start, stop, step)
-      case ArraySort(_) =>
+      case ArraySort(_, ascending) =>
         val IndexedSeq(a: IR) = newChildren
-        ArraySort(a)
+        ArraySort(a, ascending)
```

Comment:
Does `ascending` need to be counted as a child?

Reply:
Yup. At first `ascending` was a compile-time value. I missed this when I changed it.